pages/common/ping.md

```diff
@@ -1,6 +1,7 @@
 # ping
 
 > Send ICMP ECHO_REQUEST packets to network hosts.
+> More information: <https://man.openbsd.org/ping>.
```

Comment:
Even tough `ping` originally comes from `BSD`, the version we currently use on Linux doesn't. So we should use a more general link
```suggestion
> More information: <https://manned.org/ping>.
```

Reply:
Thanks @marchersimon for the correction.